client/src/components/layout/MovieTile.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import '../../assets/scss/styles.scss'
```

Comment:
styling is imported in the `App.js` component and makes the styles available to all your React components (that are rendered by App, which should be all components as App is the starting point of all pages)